services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerClusterConfig.java

```diff
@@ -1767,6 +1770,10 @@ public int getDanglingTopicOccurrenceThresholdForCleanup() {
     return danglingTopicOccurrenceThresholdForCleanup;
   }
 
+  public boolean getUpdateRealTimeTopic() {
```

Comment:
Can we update the name of this method to `isHybridStorePartitionCountUpdateEnabled()`